pkg/resource/layer_version/hooks.go

```diff
@@ -0,0 +1,80 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package layer_version
+
+import (
+	"context"
+	"sort"
+
+	// "github.com/Jeffail/gabs/v2"
+	svcapitypes "github.com/aws-controllers-k8s/lambda-controller/apis/v1alpha1"
+	svcsdk "github.com/aws/aws-sdk-go/service/lambda"
+)
+
+var (
+	_ = &svcsdk.Lambda{}
+	_ = &svcapitypes.LayerVersion{}
+)
```

Comment:
Do you need declarations here?

Reply:
If I remove them it creates error after in the code

Reply:
Done